pom.xml

```diff
@@ -629,18 +628,6 @@
         <version>${version.com.github.tdomzal.junit.docker}</version>
         <scope>test</scope>
       </dependency>
-      <dependency>
-        <groupId>org.apache.logging.log4j</groupId>
-        <artifactId>log4j-core</artifactId>
-        <version>${version.org.apache.logging.log4j}</version>
-        <scope>test</scope>
-      </dependency>
-      <dependency>
-        <groupId>org.apache.logging.log4j</groupId>
-        <artifactId>log4j-api</artifactId>
-        <version>${version.org.apache.logging.log4j}</version>
-        <scope>test</scope>
-      </dependency>
```

Comment:
what if we remove only version and keep here the depMgmt? There is probably not needed to move it to kie-parent?

Reply:
done